libs/ui/src/components/atoms/helperTexts/helperTexts.tsx

```diff
@@ -57,3 +67,17 @@ export const ErrorText = ({ children }: SpecialHelperText) => {
     </HelperText>
   );
 };
+
+export const HelperTextFactory = ({
+  error,
+  success,
+  warning,
+  helperText,
```

Comment:
Since these arguments are mutually exclusive it will be cleaner to have two arguments. One for text and then another for type of text.

Reply:
Text could even be changed to children potentially to make this a single argument function

Reply:
Generally agree with this, but this component is trying to solve a different problem. Design had mentioned that if we have an input that has more than one helpertext on it, instead of displaying both, we render only one based on a certain priority. 

Inputting one element would require that we do similar logic (taking possibly more than one type of helper text, but displaying only one) in the component above.

Reply:
I guess it's a sort of reducer at this point. Could change the name from factory to reducer.